ballerina/client.bal

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+# Consists of APIs to integrate with Avro Schema Registry.
+public isolated class Client {
+
+    private handle schemaRegistryClient;
+
+    # Gets invoked to initialize the `connector`.
+    #
+    # + config - The configurations to be used when initializing the `connector`
+    # + return - An `cregistry:Error` if connector initialization failed
+    public isolated function init(ConnectionConfig config) returns Error? {
```

Comment:
Can't we mark `config` as included record param [1] ? That is the best practice for passing the config values in the `init`.

[1] - [https://ballerina.io/learn/by-example/included-record-parameters/](https://ballerina.io/learn/by-example/included-record-parameters/)